tests/router.php

```diff
@@ -33,4 +33,11 @@
 define('GLPI_CONFIG_DIR', __DIR__);
 define('GLPI_PICTURE_DIR', __DIR__ . '/files/_pictures');
 
+// Avoid warnings because of missing globals
+$DEBUG_SQL = [
+    'queries' => [],
+    'errors'  => [],
+    'times'   => [],
+ ];
+
```

Comment:
When running webtests, this was generating a lot of warnings in the "php -S" server used to run the web tests.

@cedric-anne 
Maybe any warning or error emitted by this web server should mark the tests as failed (they are currently ignored) ?
No idea if this is possible.